docs/getting_started.md

```diff
@@ -0,0 +1,191 @@
+# How do I work in Reaction Force?
+
+We're going to take a component, and make it fit inside Reaction Force. This doc is based 
+on [this PR](https://github.com/artsy/reaction-force/pull/176).
+
+## Simple
+
+```jsx
+import React, { Component } from "react"
+
+class IconImageSet extends Component {
+  render() {
+    return (
+      <svg id="imageset" data-name="imageset" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 44 44">
+        <path d="M6760.67,551.72h-32v-32h32v32Zm-30-2h28v-28h-28v28Z" transform="translate(-6728.67 -507.72)" />
+        <polygon points="38 37 36 37 36 8 7 8 7 6 38 6 38 37" />
+        <polygon points="44 31 42 31 42 2 13 2 13 0 44 0 44 31" />
+      </svg>
+    )
+  }
+}
+export default IconImageSet
+```
+
+This is a component which returns an SVG. To move this into Reaction force, you'll need to find a place to put it. 
+
+It should go somewhere inside `src/components/` - but it will need to be a `.tsx` file. So for now, let's imagine it is
+in `src/components/icons/icon_image_set.tsx`. Make the file and paste the contents in in.
```

Comment:
Do you think we should enforce / encourage `components/CamelCase.tsx`? Since Relay 2 now requires it it seems like a good habit to get into.